pkg/tasks/thanos_ruler_user_workload.go

```diff
@@ -291,6 +278,35 @@ func (t *ThanosRulerUserWorkloadTask) create(ctx context.Context) error {
 		}
 	}
 
+	// TODO(simonpasquier): remove this step after OCP 4.16 is released.
```

Comment:
If it's not already the case, it'll be great to have a ticket to keep track of thee TODOs.

Reply:
I'll create one

Reply:
https://issues.redhat.com/browse/MON-3801